code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -786,3 +804,621 @@ nobiliumsuppression = INFINITY
+	if(cached_gases[/datum/gas/hydrogen][MOLES] > 150)
+		var produced_amount = min(5, cached_gases[/datum/gas/hydrogen][MOLES], cached_gases[/datum/gas/proto_nitrate][MOLES])
+		if(cached_gases[/datum/gas/hydrogen][MOLES] - produced_amount <0)
+			return NO_REACTION
+		cached_gases[/datum/gas/hydrogen][MOLES] -= produced_amount
+		cached_gases[/datum/gas/proto_nitrate][MOLES] += produced_amount * 0.5
+		energy_released -= produced_amount * 2500
+		if(energy_released)
+			var/new_heat_capacity = air.heat_capacity()
+			if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+				air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
+		return REACTING
+
+/datum/gas_reaction/proto_nitrate_cyrion_b_response
+	priority = 25
+	name = "Proto Nitrate Cyrion B response"
+	id = "proto_nitrate_cyrion_b_response"
+
+/datum/gas_reaction/proto_nitrate_cyrion_b_response/init_reqs()
+	min_requirements = list(
+		/datum/gas/proto_nitrate = MINIMUM_MOLE_COUNT,
+		/datum/gas/cyrion_b = MINIMUM_MOLE_COUNT,
+		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST
+	)
+
+/datum/gas_reaction/proto_nitrate_cyrion_b_response/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases
+	var/turf/open/location = isturf(holder) ? holder : null
+	if(cached_gases[/datum/gas/cyrion_b][MOLES] > MINIMUM_MOLE_COUNT)
+		var max_power = min(5, cached_gases[/datum/gas/cyrion_b][MOLES])
+		cached_gases[/datum/gas/cyrion_b][MOLES] = 0
+		explosion(location, max_power * 0.15, max_power * 0.45, max_power * 0.65, max_power, max_power* 2)
+		return REACTING
+	return NO_REACTION
+
+/datum/gas_reaction/pluox_formation
+	priority = 2
+	name = "Pluoxium formation"
+	id = "pluox_formation"
+
+/datum/gas_reaction/pluox_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/carbon_dioxide = MINIMUM_MOLE_COUNT,
+		/datum/gas/oxygen = MINIMUM_MOLE_COUNT,
+		/datum/gas/tritium = MINIMUM_MOLE_COUNT,
+		"TEMP" = 100,
+		"MAX_TEMP" = T0C
+	)
+
+/datum/gas_reaction/pluox_formation/react(datum/gas_mixture/air, datum/holder)
+	var/energy_released = 0
+	var/old_heat_capacity = air.heat_capacity()
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var produced_amount = min(5, cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/oxygen][MOLES])
+	if(cached_gases[/datum/gas/carbon_dioxide][MOLES] - produced_amount <0 || cached_gases[/datum/gas/oxygen][MOLES] - produced_amount * 0.5 <0 || cached_gases[/datum/gas/tritium][MOLES] - produced_amount * 0.01 <0)
+		return NO_REACTION
+	cached_gases[/datum/gas/carbon_dioxide][MOLES] -= produced_amount
+	cached_gases[/datum/gas/oxygen][MOLES] -= produced_amount * 0.5
+	cached_gases[/datum/gas/tritium][MOLES] -= produced_amount * 0.01
+	ASSERT_GAS(/datum/gas/pluoxium, air)
+	cached_gases[/datum/gas/pluoxium][MOLES] += produced_amount
+	ASSERT_GAS(/datum/gas/hydrogen, air)
+	cached_gases[/datum/gas/hydrogen][MOLES] += produced_amount * 0.01
+	energy_released += produced_amount * 2500
```

Comment:
Make this more clear in the changelog, it'll be much appreciated. That also seems like a good deal of heat, does it cause fires thus burning away the hydrogen?

Reply:
fixed changelog, also i tweaked the reaction to happen only if the temp is between 50 K and 273 K way lower than the min fire temp

Reply:
Doesn't this reaction produce hydrogen? Or is it too minor to capture. That's what I was referring to, you'll make some people who suffer when they touch tile atmos very happy if you mention this in the changelog

Reply:
ahhh you were talking about the h2! i'll add in the changelog that this is produced too as a byproduct

Reply:
and it should be enough to be harvested